client/client-lib/src/ln/mod.rs

```diff
@@ -39,14 +38,14 @@ use crate::utils::ClientContext;
 
 #[derive(Debug)]
 pub struct LnClient {
-    pub config: LightningModuleClientConfig,
+    pub config: LightningClientConfig,
     pub context: Arc<ClientContext>,
 }
 
 impl ClientModule for LnClient {
     const KIND: &'static str = "ln";
-    type Decoder = <LightningModule as ServerModule>::Decoder;
```

Comment:
Interesting, I hadn't even thought of the LN module in all this, but seems to make naming more consistent, so ACK.